src/routes/TraderForm/Form/TransactionDetails/index.tsx

```diff
@@ -46,9 +58,7 @@ export const TransactionDetails: React.FunctionComponent<TransactionDetailsProps
             colorToken="lavenderWeb"
             token="%"
             typographyToken="secondaryBodySmallRegular"
-            value={
-              infoPostSwap.status === 'success' ? formatNumber(infoPostSwap.value.slippage) : '--'
-            }
+            value={slippage !== null ? formatNumber(slippage, 0, 2) : '--'}
```

Comment:
seems as candidate for a selectSlippageFormatted ^_^

Reply:
[05fe837](https://github.com/Voltz-Protocol/voltz-ui/pull/587/commits/05fe837b4cb3abd9425e03bd5049214e3c9c7ea7)